### ISSUE TITLE
pyrevit for revit 2014

### ISSUE THREAD

**Issue by welly76** [eirannejad/pyRevit#365]
Hello Sir 
I got a new project in Revit 2014 version, I have to work at the same version :( how to install pyrevit in Revit 2014, please advice and help me.
thanks

**Comment by eirannejad** [comment]
Hey. Go to `%appdata%/Autodesk/Revit/Addins` directory and copy the `pyRevit.addin` file from other versions into the `2014` folder. Then reload Revit and it should load pyRevit with no issues. I'll add it to the list in the pyRevit settings as well.

- [ ] Add old versions to the supported addin list in settings

**Comment by welly76** [comment]
thank you very much

**Comment by welly76** [comment]
did not work :( error

**Comment by eirannejad** [comment]
Give me more info please. What's the error?

**Comment by welly76** [comment]
![photo_2018-08-11_19-31-58](https://user-images.githubusercontent.com/37000707/43994001-6230e570-9d9e-11e8-9e92-2cb43a58a85e.jpg)

**Comment by welly76** [comment]
hello sir 
I'm using Revit in 2014 with big projects  when getting revision tasks got this msgs "
)
![revison task](https://user-images.githubusercontent.com/37000707/45324221-2bb69100-b556-11e8-8c4a-0882f6ee8af3.jpg)
please advise
regards

**Comment by welly76** [comment]
most of the add-ons are not working like 2018

**Comment by eirannejad** [comment]
@welly76 Yeah 2014 is old and the Revit API has improved a lot from then. Unfortunately I don't have access to Revit 2014 anymore :/

**Comment by welly76** [comment]
thanks, I will update my projects
